docs/src/index.md

```diff
@@ -45,20 +45,20 @@ julia> problem = MISProblem(g)
 MISProblem(20)
 
 # select the branching strategy
-julia> branching_strategy = OptBranchingStrategy(TensorNetworkSolver(), IPSolver(), EnvFilter(), MinBoundarySelector(2), D3Measure())
-OptBranchingStrategy
+julia> branching_strategy = BranchingStrategy(TensorNetworkSolver(), IPSolver(), EnvFilter(), MinBoundarySelector(2), D3Measure())
+BranchingStrategy
     ├── table_solver - TensorNetworkSolver()
     ├── set_cover_solver - IPSolver(10)
     ├── pruner - EnvFilter()
     ├── selector - MinBoundarySelector(2)
     └── measure - D3Measure()
 
 
-julia> config = SolverConfig(MISReducer(), branching_strategy, MISSize)
+julia> config = SolverConfig(MISReducer(), branching_strategy, Int)
```

Comment:
I think `MISSize` is more clear, how should someone know what is represented by `Int`?